src/ServiceControl/Infrastructure/Settings/Settings.cs

```diff
@@ -526,6 +526,14 @@ int GetDataSpaceRemainingThreshold()
             return threshold;
         }
 
+
+        DataStoreType GetDataStoreType()
+        {
+            var value = SettingsReader<string>.Read("DataStoreType", "DataStoreType.RavenDb");
```

Comment:
This default wouldn't parse. It needs to be the string value of the enum, e.g. `RavenDb`.

```suggestion
            var value = SettingsReader<string>.Read("DataStoreType", "RavenDb");
```